### ISSUE TITLE
[Accessibility: Error Identification] Device Ids input field not getting focused on validation failure in Add FCP SAN Adapter.

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/gingers390x#45]
Device Ids input field not getting focused on validation failure in Add FCP SAN Adapter.

![fcp san](https://cloud.githubusercontent.com/assets/13199397/13984470/5bc6839c-f11c-11e5-8458-bc99266884b6.jpg)
